modules/prometheus/main.tf

```diff
@@ -0,0 +1,135 @@
+data "aws_subnet" "selected" {
+  id = "${var.subnet_id}"
+}
+
+data "aws_vpc" "selected" {
+  id = "${data.aws_subnet.selected.vpc_id}"
+}
+
+resource "aws_instance" "prometheus" {
+  ami           = "${var.ami_id}"
+  instance_type = "${var.instance_type}"
+  key_name      = "${var.ssh_key_name}"
+  subnet_id     = "${var.subnet_id}"
+
+  user_data = "${data.template_file.user_data.rendered}"
+
+  associate_public_ip_address = "${var.associate_public_ip_address}"
+  vpc_security_group_ids      = ["${aws_security_group.prometheus.id}"]
+  iam_instance_profile        = "${aws_iam_instance_profile.prometheus.name}"
+  tags                        = "${merge(var.tags, map("Name", "${var.name}"))}"
+  volume_tags                 = "${merge(var.tags, map("Name", "${var.name}"))}"
+
+  root_block_device {
+    volume_type = "gp2"
+    volume_size = "${var.root_volume_size}"
+  }
+}
```

Comment:
Any way forward to HA-ize / Nomad-ize the Prometheus? The last I know of is [Thanos](https://github.com/improbable-eng/thanos).

Reply:
Looked into it. Can't find a good way to do that. Will be possible once RDS allows installing the right extensions needed.